unlock-app/src/components/lock/UnlockedFlag.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components'
+import React from 'react'
+import { connect } from 'react-redux'
+import { RoundedLogo } from '../interface/Logo'
+import { Colophon } from './Overlay'
+import { mapStateToProps } from './ShowUnlessUserHasKeyToAnyLock'
+
+export class UnlockedFlag extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      hidden: false,
+    }
+  }
+
+  componentDidMount() {
+    setTimeout(() => this.setState({ hidden: true }), 2000)
```

Comment:
Let's put the 200 into `constants.js` please.

Reply:
ok